.github/workflows/images-legacy.yaml

```diff
@@ -0,0 +1,122 @@
+name: Image CI Build
+
+on:
+  pull_request_target:
```

Comment:
Should we, in addition to the CODEOWNERs change, also add some comments somewhere here, mentioning that this workflow will have access to the credentials, therefore should not invoke Makefiles and such?